_includes/page-types/welcome.html

```diff
@@ -5,13 +5,22 @@ <h2 class="page-heading">{{ page.title }}</h2>
       <div class="col-md-8">
         {{ page.content }}
         
-        <img src="/assets/img/kerri-signature.svg" alt="Kerri Hoffman's signature" class="signature" />
+        <img src="/assets/img/kerri-signature.png" alt="Kerri Hoffman's signature" width="250" height="92" class="signature" />
       </div>
+
+      {%- if page.image -%} 
       <div class="col-md-4">
         <figure class="page-image">
           {% picture {{ page.image }} --alt {{ page.image-alt }}  %}
           <p class="caption fs-6">{{ page.image-caption }}</p>
         </figure>
+      </div>
+      {%- endif -%}
+    </div>  
+    <div class="row mt-4">
+      {%- if page.content2 -%}
+        {{ page.content2 }}
+      {%- endif -%}
```

Comment:
Added so I could sneak a paragraph underneath the signature.

Reply:
Shouldn't the row div be inside the condition? right now you are always getting an empty wrapper that adds margin somewhere.

Reply:
Fixed!